news/_posts/2020-01-07-critical-security-vulnerability-in-prestashop-modules.md

```diff
@@ -93,3 +95,57 @@ Modules and themes by other vendors may be vulnerable as well. Expect updates to
 
 A very special thank you to Francis Ladeuil who first reported this issue.
 
+## FAQ
+
+You can find answers to some frequent questions asked these past days.
+
```

Comment:
This is awful but you can add CSS instead of using style on each element:

```html
<style type="text/css">
details { padding: .5em .5em 0; }
details summary { margin: -.5em -.5em 0; padding: .5em; }
</style>
```

Reply:
I would say let's take some time later to add proper css properties in the stylesheets for these elements so that we could use them confidently in the future.